.gitignore

```diff
@@ -21,3 +21,5 @@ visor
 build/.*
 support/tools/bin
 
+# Deployment
+deployment/.env
```

Comment:
We will put the `.env` file in the deployment folder because of docker-compose usage, therefore we should add the `gitignore` here  😄 .

Reply:
I change it to `*.env`.

Reply:
Simply doing `.env` would ignore it in all paths instead of `*.env` which would only ignore in the root path. If we do hyperspecific paths like so, we could face the same problem again where the secrets are revealed.